<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Salarios</class>
 <widget class="QMainWindow" name="Salarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Salarios</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="inNombre">
            <property name="styleSheet">
             <string notr="true">
font: 16pt &quot;Storytime&quot;;</string>
            </property>
            <property name="text">
             <string>Nombre:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit">
            <property name="placeholderText">
             <string>Ingrese el Nombre del Obrero</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="inHoras">
            <property name="styleSheet">
             <string notr="true">font: 16pt &quot;Storytime&quot;;</string>
            </property>
            <property name="text">
             <string>Horas:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;Storytime&quot;;</string>
         </property>
         <property name="title">
          <string>Jornada</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="inMatutina">
            <property name="text">
             <string>Matutina</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="inVespertina">
            <property name="text">
             <string>Vespertina</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="inNocturna">
            <property name="text">
             <string>Nocturna</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>180</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="BtnCalcular">
            <property name="styleSheet">
             <string notr="true">font: 800 10pt &quot;Rockwell Extra Bold&quot;;</string>
            </property>
            <property name="text">
             <string>&amp;Calcular</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>24</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="outDatos">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Resultado de los CÃ¡lculos</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo_2"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar_3"/>
    <addaction name="action_Calcular_2"/>
    <addaction name="action_Salir_3"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAcerca_de_Salarios"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAyuda"/>
  </widget>
  <action name="actionAcerca_de_Salarios">
   <property name="text">
    <string>Acerca de Salarios</string>
   </property>
  </action>
  <action name="actionCalculos">
   <property name="text">
    <string>Calculos</string>
   </property>
  </action>
  <action name="actionSalida">
   <property name="text">
    <string>Salida</string>
   </property>
  </action>
  <action name="actionNuevo">
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="action_Salir">
   <property name="text">
    <string>&amp;Salir</string>
   </property>
  </action>
  <action name="action_Calcular">
   <property name="text">
    <string>&amp;Calcular</string>
   </property>
  </action>
  <action name="actionNuevo_2">
   <property name="text">
    <string>&amp;Nuevo</string>
   </property>
   <property name="shortcut">
    <string>CapsLock, CapsLock</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>A&amp;brir</string>
   </property>
  </action>
  <action name="actionGuardar_3">
   <property name="text">
    <string>&amp;Guardar</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="action_Calcular_2">
   <property name="text">
    <string>&amp;Carcular</string>
   </property>
  </action>
  <action name="action_Salir_3">
   <property name="text">
    <string>&amp;Salir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
